client/apps/plugin-status/state/actions.js

```diff
@@ -10,21 +10,9 @@ import { translate } from 'i18n-calypso';
 import * as api from 'api';
 // from Calypso
 import { successNotice, errorNotice } from 'state/notices/actions';
+import { PLUGIN_STATUS_DEBUG_TOGGLE, PLUGIN_STATUS_LOGGING_TOGGLE, SERVICE_DATA_REFRESH } from './action-types';
 
-export const PLUGIN_STATUS_DEBUG_TOGGLE = 'PLUGIN_STATUS_DEBUG_TOGGLE';
```

Comment:
Having the action types inside the `actions` file is bad, because it forces the reducer to also import the actions (whereas the actions and the reducer should not be aware of each other)

Moved these to a new `action-types.js` file